### ISSUE TITLE
Error passed from RabbitMQ has no meaningfull information

### ISSUE THREAD

**Issue by TrejGun** [nestjs/nest#6191]
## Current behavior
I ran out of space in my docker and was unable to connect to RabbitMQ
because of RabbitMQ's error 
```
➜  ~ docker run -p 5672:5672 -p 15672:15672 rabbitmq:management

01:03:58.550 [error] Failed to write to cookie file '/var/lib/rabbitmq/.erlang.cookie': enospc

01:03:59.368 [error] Too short cookie string
```

However error printed by NestJS has no any usefull information
```
[Nest] 71853   - 1/19/2021, 9:01:44 AM [Server] 
```
I inspected error object passed to logger and it looks like this
```
{
  context: 'Server',
  stack: [ '' ],
  err: OperationalError: connect ECONNREFUSED 127.0.0.1:5672
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)
  From previous event:
      at Object.connect (~/node_modules/amqplib/channel_api.js:6:18)
      at ~/node_modules/amqp-connection-manager/lib/AmqpConnectionManager.js:179:31
      at processTicksAndRejections (internal/process/task_queues.js:93:5) {
    cause: Error: connect ECONNREFUSED 127.0.0.1:5672
        at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
      errno: -61,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 5672
    },
    isOperational: true,
    errno: -61,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '127.0.0.1',
    port: 5672
  },
  level: 'error',
  message: undefined,
  timestamp: '2021-01-19T01:01:44.505Z',
  [Symbol(level)]: 'error',
  [Symbol(splat)]: [ { context: 'Server', stack: [Array], err: [OperationalError] } ]
}
```

## Expected behavior
I'd like to see some meaningfull error description 

at least like with postgress 
```
[Nest] 72537   - 1/19/2021, 9:22:20 AM [TypeOrmModule] Error: connect ECONNREFUSED 127.0.0.1:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)
```

## Possible Solution
unknown

## Environment

<pre><code>
Nest version: 7.6.5

 
For Tooling issues:
- Node version: v14.15.3
- Platform:  mac big sur

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Would you like to create a PR for this issue?

**Comment by TrejGun** [comment]
I can try

**Comment by kamilmysliwiec** [comment]
This should be fixed in `@nestjs/common@7.6.6`

**Comment by TrejGun** [comment]
Thanks!
